src/ensembl/src/content/app/species-selector/components/species-commit-button/SpeciesCommitButton.tsx

```diff
@@ -17,27 +17,35 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+import useSpeciesSelectorAnalytics from 'ensemblRoot/src/content/app/species-selector/hooks/useSpeciesSelectorAnalytics';
```

Comment:
let's clear out the `ensemblRoot` in this PR

Reply:
That's done